### PR TITLE
FFTW: Auto-disable quad precision on POWER and ARM

### PR DESCRIPTION
**Pull request by geimer**
Per @boegel's request: This is a wrapper PR around https://github.com/hpcugent/easybuild-easyblocks/pull/1102.

### CODE REVIEW

File: easybuild/easyblocks/f/fftw.py [hunk 1]
```diff
@@ -116,6 +116,11 @@ def __init__(self, *args, **kwargs):
                     self.log.info("Enabling use of %s (should be supported based on CPU features)", flag.upper())
                     setattr(self, flag, True)
 
+        # Auto-disable quad-precision on ARM and POWER, as it is unsupported
+        if self.cfg['with_quad_prec'] and arch in [AARCH32, AARCH64, POWER]:
```
Comment (boegel) [inline easybuild/easyblocks/f/fftw.py]: `s/arch/cpu_arch/`?

@geimer was this tested? :)

Comment (geimer) [inline easybuild/easyblocks/f/fftw.py]: @boegel I explicitly mentioned that this was **not** tested.  And yes, you are right w.r.t. `cpu_arch`.

Comment (boegel) [inline easybuild/easyblocks/f/fftw.py]: Well, maybe in the PR to Dylan's repo, but I forget easily. ;-)

I'll make sure to give this a proper test once you've changed this to `cpu_arch`.

**Comment by geimer** [comment]
@boegel Successfully tested on POWER, no access to ARM right now...

**Comment by boegel** [comment]
rested on x86-64, no impact, so good to go, thanks @geimer!

**Pull request merged**